docs/users.md

```diff
@@ -97,14 +102,16 @@ type: Opaque
 stringData:
   MONGODB_BACKUP_USER: backup
   MONGODB_BACKUP_PASSWORD: backup123456
+  MONGODB_DATABASE_ADMIN_USER: databaseAdmin
+  MONGODB_DATABASE_ADMIN_PASSWORD: databaseAdmin123456
   MONGODB_CLUSTER_ADMIN_USER: clusterAdmin
   MONGODB_CLUSTER_ADMIN_PASSWORD: clusterAdmin123456
   MONGODB_CLUSTER_MONITOR_USER: clusterMonitor
   MONGODB_CLUSTER_MONITOR_PASSWORD: clusterMonitor123456
   MONGODB_USER_ADMIN_USER: userAdmin
   MONGODB_USER_ADMIN_PASSWORD: userAdmin123456
-  PMM_SERVER_USER: pmm
-  PMM_SERVER_PASSWORD: supa|^|pazz
+  PMM_SERVER_USER: admin
+  PMM_SERVER_PASSWORD: admin
 ```
```

Comment:
@hors should we leave pmm passwords here?

Reply:
So, we need to have it but we need to mention that we suggest using API KEY (and maybe add the link to PMM doc because PMM team also suggest using API KEY  instead of user/password)

Reply:
Already done

Reply:
I believe we need to mention api keys also in this docs otherwise people can just copy/past and continue to use user/pass insted of api-key. @hors what do you think?

Reply:
Yes, we need to add new  api-key

Reply:
There is a warning (line 70) exactly after the table, it links to our api key doc and says that password-based variant is deprecated for pmm. Placing the key in the !ist of users looks !ittle bit strange: it's not a user, and has nothing human-readable... So there is a warning instead of an actual api key...